### PR TITLE
Handle ocm roles

### PR DESCRIPTION
**Pull request by arivepr**
Addressing [this jira card](https://issues.redhat.com/browse/RHCLOUD-20404)

7184E220-50F1-4F3F-A1DF-34822463F85B

### CODE REVIEW

File: src/smart-components/role/role.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ import './role.scss';
 const Role = ({ onDelete }) => {
   const history = useHistory();
   const [isDropdownOpen, setDropdownOpen] = useState(false);
+  const [isOCMSystem, setIsOCMSystem] = useState(false);
```
Comment (Hyperkid123) [inline src/smart-components/role/role.js:25]: I would probably change the name of this variable. Technically the same can apply for other roles than just OCM.

File: src/smart-components/role/role-permissions.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ const removeModalText = (permissions, role, plural) =>
     </p>
   );
 
-const Permissions = () => {
+const Permissions = (isOCM) => {
```
Comment (Hyperkid123) [inline src/smart-components/role/role-permissions.js:56]: Again not just OCM

**Review by Hyperkid123 — COMMENTED** [review]
Looks good. Just a little naming nitpick.

File: src/smart-components/role/role.js (hunk 1 above)
Comment (arivepr) [reply src/smart-components/role/role.js:25]: hmmm,  nonPermissionAddingRole?

**Comment by Hyperkid123** [comment]
@arivepr can you fix the tests? And add another case. It looks like you are now trying to run the add permissions on a role that matches the criteria.

**Review by aneelac22 — APPROVED** [review]
LGTM!

**Pull request merged**